### DISCUSSION TITLE
Projections cli not working as expected in Marten 6

### DISCUSSION THREAD

**Discussion by nmowatt** [JasperFx/marten#2566] [Q&A]
Regardless of what command I'm running, e.g:

```
dotnet run -- projections -i --rebuild
dotnet run -- projections -i
dotnet run -- projections --list
```

It seems to immediately start running the projections. I would expect to go into interactive mode or being listed the different projections I have.

Output:
```
Building...
info: Marten.Events.Daemon.AsyncProjectionHostedService[0]
      Started HighWaterAgent
info: Marten.Events.Daemon.AsyncProjectionHostedService[0]
      Starting projection agent for 'Booking:All'
info: Marten.Events.Daemon.AsyncProjectionHostedService[0]
      Projection agent for 'Booking:All' has started from sequence 80 and a high water mark of 80
```

**Comment by jeremydmiller** [comment]
@nmowatt Hey, before we call this a bug, what's your Marten setup for the daemon like? And how are you applying Oakton?

**Comment by nmowatt** [comment]
I'm not using Oakton explicitly in any way, so not sure how that will be applied.
My setup for the async daemon is just:
```
.AddAsyncDaemon(DaemonMode.Solo);
```

Does that answer your question @jeremydmiller ?

**Comment by nmowatt** [comment]
Did you try to use the projections cli in Marten 6 @jeremydmiller ? E.g. projections --list

**Comment by jeremydmiller** [comment]
@nmowatt "I'm not using Oakton explicitly in any way, so not sure how that will be applied."

-- Oakton is what relays the command line options to the projections command. 

See the docs here: https://martendb.io/configuration/cli.html

So at no point are you even executing the projections command. What you're seeing is the application just start up normally.
